x-pack/solutions/search/plugins/search_homepage/public/components/connect_to_elasticsearch_side_panel.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiSplitPanel,
+  EuiCard,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiText,
+  EuiLink,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../hooks/use_kibana';
+
+const UPLOAD_FILE_URL = '/app/home#/tutorial_directory/fileDataViz';
+
+export const ConnectToElasticsearchSidePanel = () => {
+  const { http } = useKibana().services;
+
+  return (
+    <EuiSplitPanel.Outer hasShadow={false} color="subdued">
+      <EuiSplitPanel.Inner>
+        <EuiCard
+          textAlign="left"
+          titleSize="xs"
+          title={i18n.translate('xpack.searchHomepage.connectToElasticsearch.uploadFileTitle', {
+            defaultMessage: 'Upload a file',
+          })}
+          description={
+            <FormattedMessage
+              id="xpack.searchHomepage.connectToElasticsearch.uploadFileDescription"
+              defaultMessage="Upload your file, analyze its data, and import the data into an Elasticsearch index."
```

Comment:
cc: @leemthompo  copy

Reply:
Per other suggestion I think this should be consolidated, but if not I think it should be shortened:
```suggestion
              defaultMessage="Analyze and import data from a file."
```